app/views/entities/index.html.erb

```diff
@@ -0,0 +1,55 @@
+<header>
+    <div>
+        <%= link_to image_tag('back.png'), groups_path, id: "back-icon" %>
+    </div>
+    <h4>
+        <%= @group.name %> <br/> Transaction
+    </h4>
+    <div>
+        <%= button_to "Logout", destroy_user_session_path, method: :delete %>
+    </div>
+</header>
+
+  <div class="gift-card">
+    <p>Gift Card</p>
+    <h3 class="money">$45.00</h3>
+  </div>
+
+<div class="transaction" >
+    <div class="category-card mb-3" >
+        <%= image_tag url_for(@group.icon), alt: "group icon" %>
+        <div class="inner-card" >
+            <div class="inner-card-info">
+                <h3> <%= @group.name %> </h3>
+                <span style="opacity: 0 ;" >
+                    .
+                </span>
+            </div>
+            <h4> <%= @group.created_at.strftime('%B %d, %Y %I:%M %p') %> </h4>
+        </div>
+    </div>
+
+    <div class="transaction-group">
+        <% @entities.each do |entity| %>
+            <div class="transaction-card card mb-3">
+                <div class="card-body">
+                    <h4 class="card-title"><%= entity.name %></h4>
+                    <h6 class="card-subtitle mb-2 text-muted">Amount: $<%= entity.amount %></h6>
+                    <p class="card-text">Created at: <%= entity.created_at.strftime('%B %d, %Y %I:%M %p') %></p>
+                </div>
+            </div>
+        <% end %>
+        <div class="transaction-card card mb-3" style="background-color: #007BFF; color: #fff;">
+            <div class="card-body">
+                <h4 class="card-title">Payment</h4>
+                <h4 class="card-subtitle mb-2">Total: $<%= @group.entities.sum(&:amount) %></h4>
+            </div>
+        </div>
+    </div>
+</div>
+
+<footer>
+    <div class="action">
+        <%= link_to "add new transaction", new_group_entity_path(group_id: @group.id) %>
```

Comment:
- [ ] As the previous reviewer mentioned, kindly use the right color for the button. You can see the color you used was turned off by me, and it is not the same as #5fb523` which you included but its not used 

### View:
![image](https://github.com/Cosmosha/BudgetPal/assets/50990925/20acf344-2843-41b2-8752-434dd516d1fc)